github/table_github_repository_sbom.go

```diff
@@ -0,0 +1,103 @@
+package github
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableGitHubRepositorySbom() *plugin.Table {
+	return &plugin.Table{
+		Name:        "github_repository_sbom",
+		Description: "SBOM from a repository.",
```

Comment:
```suggestion
		Description: "Get the software bill of materials (SBOM) for a repository.",
```

Description partially taken from https://docs.github.com/en/rest/dependency-graph/sboms?apiVersion=2022-11-28.